Automattic/wp-calypso
# Add/updated creds flow calypso

#### Changes proposed in this Pull Request

* This patch implements stream processing of `application/x-ndjson responses`. It is a port of the code we used in the `wpcom-xhr-request` library (see [PR here](https://github.com/Automattic/wpcom-xhr-request/pull/31)). We've also brought the advanced-credentials component from jetpack-cloud.

#### Testing instructions



* To test this PR you'll need to apply this diff in your wpcom sandbox: D75408-code.

* Then, change your `/etc/hosts` file to point to your wpcom sandbox.

* Start calypso locally: `yarn start`

* Create a JN site and add a Backup plan to it.

* Then go to http://calypso.localhost:3000/settings/jetpack/<JN_site_url>#credentials

* And test the credentials flow with valid and invalid credentials. It should behave the same way as in jetpack-cloud.
